app/routes.py

```
@@ -1,8 +1,50 @@
-from app import app
-from flask_cors import CORS
+from flask import jsonify, request, Blueprint
```

Ruff (0.11.9)

`flask.request` imported but unused

Remove unused import: `flask.request`

(F401)